test/DagligSkaevTest.java

```diff
@@ -0,0 +1,53 @@
+import static org.junit.Assert.assertEquals;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+import org.junit.Test;
+
+import ordination.DagligSkaev;
+import ordination.Laegemiddel;
+
+public class DagligSkaevTest {
+
+    Laegemiddel lm1 = new Laegemiddel("CBD", 2, 4, 6, "ML");
```

Comment:
```suggestion
    Laegemiddel lm1 = new Laegemiddel("CBD", 2, 4, 6, "ml");
```
Small thing: This is capitalized when the others are not